api/src/queries/occurrence/occurrence-create-queries.ts

```diff
@@ -0,0 +1,107 @@
+import { SQL, SQLStatement } from 'sql-template-strings';
+import { PostOccurrence } from '../../models/occurrence-create';
+import { getLogger } from '../../utils/logger';
+import { parseUTMString } from '../../utils/spatial-utils';
+
+const defaultLog = getLogger('queries/occurrence/occurrence-create-queries');
+
+export const postOccurrenceSubmissionSQL = (surveyId: number): SQLStatement | null => {
```

Comment:
let's add tests for these two functions in the queries file, cause it is low hanging test coverage that can be added in now